manifests/params.pp

```diff
@@ -50,9 +50,9 @@
     default:  { fail("Unsupported OS family: ${facts['os']['family']}") }
   }
 
-  if (versioncmp($::puppetversion, '4.0.0') >= 0) and ($::rubysitedir =~ /\/opt\/puppetlabs\/puppet/) {
+  if (versioncmp($facts['puppetversion'], '4.0.0') >= 0) and ($facts['ruby']['sitedir'] =~ /\/opt\/puppetlabs\/puppet/) {
     $lens_dir = '/opt/puppetlabs/puppet/share/augeas/lenses'
-  } elsif (defined('$is_pe') and str2bool("${::is_pe}")) { # lint:ignore:only_variable_string
+  } elsif (defined('$is_pe') and str2bool("${facts['is_pe']}")) { # lint:ignore:only_variable_string
```

Comment:
I'm not sure if it's still a thing at all, but I think this is better:
```suggestion
  } elsif ('is_pe' in $facts and str2bool("${facts['is_pe']}")) { # lint:ignore:only_variable_string
```

Reply:
Good catch! Done!